renderdoc/driver/gl/gl_resources.cpp

```diff
@@ -543,7 +547,7 @@ GLenum GetSizedFormat(const GLHookSet &gl, GLenum target, GLenum internalFormat,
   }
 
   GLint red, depth, stencil;
-  if(HasExt[ARB_internalformat_query2] && gl.glGetInternalformativ)
+  if(gl.glGetInternalformativ)
```

Comment:
Since you're on GLES it looks like the glGetInternalformativ only supports the queries from ARB_internalformat_query (without the 2). This only supports querying `pname == NUM_SAMPLE_COUNTS` or `pname == SAMPLES`. There's no equivalent GLES extension for `ARB_internalformat_query2`.

I don't know your implementation, maybe it supports more queries than the spec says (if it's overlapping with a GL implementation maybe without properly enforcing API limitations?) but we can't rely on that.